src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/P11Key.java

```diff
@@ -412,7 +416,7 @@ static PrivateKey privateKey(Session session, long keyID, String algorithm,
         if (keySensitive && (SunPKCS11.mysunpkcs11 != null) && "RSA".equals(algorithm)) {
             try {
                 byte[] key = SunPKCS11.mysunpkcs11.exportKey(session.id(), attributes, keyID);
-                RSAPrivateKey rsaPrivKey = RSAPrivateCrtKeyImpl.newKey(key);
+                RSAPrivateKey rsaPrivKey = RSAPrivateCrtKeyImpl.newKey(KeyType.RSA, "PKCS#1", key);
```

Comment:
The second argument must be `"PKCS#8"` (see the [head stream](https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/openj9/src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/P11Key.java#L437)).

Reply:
Fixed